app/lib/concerns/controller_navigation.rb

```diff
@@ -24,16 +24,22 @@ def next
     seek(controllers_until_end)
   end
 
+  def prev
+    return unless index&.nonzero?
+
+    controllers_to_beginning = controllers[0..index - 1].reverse
+    seek(controllers_to_beginning)
+  end
+
   private
 
   def index
-    controllers.index(@current_controller.class)
+    controllers.index(current_controller.class)
```

Comment:
The way that Rspec implements mock controllers means that current_controller.class on a mocked controller returns an instance of a class that looks like this `<Class#1239123>` instead of the class name as a const. As such get the tests for prev_path working in the controller tests, I needed to stub current_controller, and I cant stub an instance variable, so I turned it into an accessor method.

Reply:
The alternatives were to:
- not test the method
- use eval(current_controller.class.name) to turn the stringified class name back into a const
- refactor a lot of this controller navigation code that I don't fully understand the implications of.

So this seemed like the best option.